src/Dont/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont\Exception;
+
+use Throwable;
+
+/**
+ * Marker interface for exceptions thrown by the Dont package
+ */
+interface ExceptionInterface extends Throwable
```

Comment:
It's a marker interface - provides generic "group by" and catch-all for the package, should anybody want to do that.